modules/applyPresentersToData.js

```diff
@@ -1,6 +1,15 @@
 const righto = require('righto');
 const evaluate = require('./evaluate');
 
+function transformArraysProperty (schema, record) {
+  Object.keys(schema).forEach(schemaKey => {
```

Comment:
This is mutating the record, it would be nice to instead reduce over the schema and build a new result.

Reply:
Few more lines now, but worth it for the clarity